Package.swift

```diff
@@ -33,6 +37,12 @@ let package = Package(
                 .target(name: "UtilityBeltNetworking"),
             ]
         ),
+        .target(
+            name: "ShamTestHelpers",
```

Comment:
I'd prefer we update the new target names to be more explicit about what they are intended for. 

For example, this one could be `ShamXCTestSupport` or something. I'm even fine with introducing a convention like adding an underscore for extension modules, like `Sham_XCTestSupport`, especially given that these are primarily going to be used for incompatibilities for `XCTest` and so on.